multiqc/modules/qualimap/QM_BamQC.py

```diff
@@ -138,15 +138,20 @@ def parse_coverage(self, f):
         log.debug("Couldn't parse contents of coverage histogram file {}".format(f['fn']))
         return None
 
-    # Find median without importing anything to do it for us
+    # Find median and mean without importing anything to do it for us
     num_counts = sum(d.values())
     cum_counts = 0
+    total_cov = 0
     median_coverage = None
+    mean_coverage = None
     for thiscov, thiscount in d.items():
         cum_counts += thiscount
+        total_cov += thiscov*thiscount
         if cum_counts >= num_counts/2:
             median_coverage = thiscov
             break
+    mean_coverage = total_cov / num_counts
+    self.general_stats_data[s_name]['mean_coverage' = mean_coverage
```

Comment:
Missing a square bracket here:

```suggestion
    self.general_stats_data[s_name]['mean_coverage'] = mean_coverage
```